src/wing/CCPACSWingSegment.cpp

```diff
@@ -1041,6 +981,15 @@ double CCPACSWingSegment::GetReferenceArea(TiglSymmetryAxis symPlane) const
     return quadrilateral_area(innerTepProj, outerTepProj, outerLepProj, innerLepProj);
 }
 
+PNamedShape CCPACSWingSegment::GetLoft(TiglShapeModifier mod) const
+{
+    // TODO
```

Comment:
@bernhardmgruber What has to be done here? Could you create an issue for it? Else, remove the comments.